packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1892,4 +1881,51 @@ async function revokeCertifiedAttribute(
   } satisfies Tenant;
 }
 
+function validateAddress(address: string): string {
+  // Here I am removing the non-printing control characters
+  const removeNonPrintingcontrolCharacters = address.replace(
+    // eslint-disable-next-line no-control-regex
+    /[\x00-\x1F\x7F]/g,
+    ""
+  );
+
+  // Here I am removing the extra spaces and tabs
+  const removeExtraSpace = removeNonPrintingcontrolCharacters
+    .replace(/\s+/g, "")
+    .trim();
+
+  // Here I am removing strange characters or special symbols
+  const sanitizedMail = removeExtraSpace
```

Comment:
I strongly recommend removing this part.
There are some allowed characters (name+surname@mail.com) that we exclude here, modifying the email passed by the user.
I think that the check on line 1904 (emailPattern) is sufficient to verify the validity of the email.